packages/@okta/vuepress-site/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/main/nodejs/auth-vs-oie.md

```diff
@@ -1,7 +1,11 @@
 
-At a high level, the classic authentication flow for the Classic Engine Authentication SDK methods and their backend API calls function similarly to the new Identity Engine SDK methods: both versions call Okta, receive a transaction object, and ultimately receive tokens after a successful authentication.
+The authentication flow for the Classic Engine Authentication SDK methods, and their backend API calls, function similarly to the Identity Engine SDK methods. Both versions call Okta, receive a transaction object, and ultimately receive tokens after a successful authentication.
```

Comment:
are the "authentication flow" and "their backend API calls" two parts of a compound subject? or is the authentication flow for the "methods and their backend API calls" ? If it's the latter, we should change it "functions"

Sorry, having a hard time parsing this sentence